tests/test-chunk.rs

```diff
@@ -38,5 +40,23 @@ async fn test_download_chunked_results() -> Result<()> {
     assert!(rows[0].column_names().contains(&"SEQ"));
     assert!(rows[0].column_names().contains(&"RAND"));
 
+    let col_types = rows[0].column_types();
+    assert!(col_types.iter().any(|(k, v)| {
```

Comment:
Since it's sorted by the index, we can make these assertions clearer, like this:

```rust
assert_eq!(column_types[0].1.snowflake_type.to_ascii_uppercase(), "FIXED");
assert!(!column_types[0].1.nullable);

assert_eq!(column_types[1].1.snowflake_type.to_ascii_uppercase(), "TEXT");
assert!(!column_types[1].1.nullable);
```

Reply:
Thanks. Fixed